src/adminPartMain/kotlin/Data.kt

```diff
@@ -147,4 +150,9 @@ data class TestTypeSummary(
 data class Risks(
     val newMethods: List<JavaMethod>,
     val modifiedMethods: List<JavaMethod>
+)
+
+@Serializable
+data class TestsToRun(
+    val tests: List<String>?
```

Comment:
Why is allow null here? I think this should be initialized by default with empty list